src/schema.ts

```diff
@@ -40,79 +49,10 @@ export type DrumType = "bassdrum" | "hihat" | "snare";
  */
 export type Channel = DrumType | "vibraphone" | "bass";
```

Comment:
Just my opinion - I'm sure there was already discussion around this: I would represent each vibraphone key, each bass string, each drum type as a separate channel.

Reply:
This was proposed, I don't quite recall the reason it was rejected, I think it was to simplify the schema?  Also since the channel configuration of the machine will be fairly static, this way is acceptable.

Reply:
Ok.